libs/vkd3d/vkd3d_main.c

```diff
@@ -344,18 +344,7 @@ static ULONG STDMETHODCALLTYPE d3d12_versioned_root_signature_deserializer_Relea
 
 static enum vkd3d_root_signature_version vkd3d_root_signature_version_from_d3d12(D3D_ROOT_SIGNATURE_VERSION version)
 {
-    switch (version)
-    {
-        case D3D_ROOT_SIGNATURE_VERSION_1_0:
-            return VKD3D_ROOT_SIGNATURE_VERSION_1_0;
-        case D3D_ROOT_SIGNATURE_VERSION_1_1:
-            return VKD3D_ROOT_SIGNATURE_VERSION_1_1;
-        case D3D_ROOT_SIGNATURE_VERSION_1_2:
-            return VKD3D_ROOT_SIGNATURE_VERSION_1_2;
-        default:
-            WARN("Unknown root signature version %#x.\n", version);
-            return 0;
-    }
+    return (enum vkd3d_root_signature_version)version;
```

Comment:
why do we even have our own enum for this at this point?

Reply:
I'm not sure why we declare internal structs which are clearly supposed to be direct aliases with the d3d ones. It might be a holdover from when vkd3d-shader was a standalone .so thing ...

Reply:
We can probably just refactor this away later, but I'd rather not do that yak shave right now.